app/src/main/java/com/isc/hermes/SignUpActivityView.java

```diff
@@ -119,8 +121,11 @@ private void sendUserBetweenActivities(User user) {
     public void onActivityResult(int requestCode, int resultCode, Intent data) {
         super.onActivityResult(requestCode, resultCode, data);
         try {
-            if (authenticationServices.containsKey(requestCode))
-                sendUserBetweenActivities(authenticator.getUserBySignInResult(data));
+            if (requestCode == REQUEST_CODE){
```

Comment:
I recommend using the '&&' operator since you have two nested ifs without any else so it would be better to have everything in one validation